src/SDKs/Search/DataPlane/Microsoft.Azure.Search.Service/Generated/Models/LanguageDetectionSkill.cs

```diff
@@ -0,0 +1,67 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Search.Models
+{
+    using Newtonsoft.Json;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// The skill detects the language of input text and reports a single
+    /// language code for every document submitted on the request. The language
+    /// code is paired with a score indicating the strength of the analysis.
```

Comment:
use "confidence" instead of "strength"?

Reply:
Fixed. However, since I copied this sentence from this link: https://docs.microsoft.com/azure/search/cognitive-search-skill-language-detection. @HeidiSteen Should we modify the document? Thanks.